src/appleseed/renderer/kernel/lighting/directlightingintegrator.cpp

```diff
@@ -233,15 +233,19 @@ void DirectLightingIntegrator::compute_outgoing_radiance_light_sampling_low_vari
     }
 
     // Add contributions from non-physical light sources only.
-    for (size_t i = 0, e = m_light_sampler.get_non_physical_light_count(); i < e; ++i)
+    if (m_light_sample_count > 0)
```

Comment:
I've found that when light sample count < 0 in studio settings direct lights are strangely brighter than usually. In such cases this variable (m_light_sample_count) gets value 0 sometimes and I thought direct light should not be added when it is 0. It fixed the brightness but I'm not sure if fix is correct.